src/main/resources/templates/users/admin/health.ftlh

```diff
@@ -83,7 +83,7 @@
                     <#list notPublicProjects as project>
                         <div class="list-group-item">
                             <a href="${routes.getRouteUrl("projects.show", project.getNamespace().getOwner(), project.getNamespace().getSlug())}">
-                                <strong>${project.namespace}</strong> <small><@spring.message "visibility.name." + project.visibility.name /></small>
+                                <strong>${project.namespace}</strong> <small><@spring.message "visibility.name." + project.getVisibility().getName() /></small>
```

Comment:
Is there a reason why you have to use the getters here instead of leaving it as is?

Reply:
Yea, it threw an exception, when you change it to what it is now, it doesnt throw it anymore
```
[Template Error] Expected a string or something automatically convertible to string (number, date or boolean), or "template output" , but this has evaluated to a method+sequence (wrapper: f.e.b.SimpleMethodModel):
==> project.visibility.name  [in template "users/admin/health.ftlh" at line 86, column 116]

----
Tip: Maybe using obj.something() instead of obj.something will yield the desired value
----

----
FTL stack trace ("~" means nesting-related):
	- Failed at: #macro message code  [in template "spring.ftl" in macro "message" at line 28, column 1]
	- Reached through: @spring.message "visibility.name." + ...  [in template "users/admin/health.ftlh" at line 86, column 78]
	~ Reached through: #nested  [in template "layout/base.ftlh" in macro "base" at line 86, column 17]
	~ Reached through: @base.base title="${message}"  [in template "users/admin/health.ftlh" at line 19, column 1]
----

Reply:
huh ok, no clue why that would be happening. I guess just leave it then.